docs/source/getting-started/first-feature.rst

```diff
@@ -224,11 +224,12 @@ You'll notice that new messages appear in the UI as soon as they are sent (due t
       :scale: 50 %
       :alt: In the create-daml-app as soon as you send the message it will appear in the "Messsaging" panel
 
+.. tip:: You completed the second part of the Getting Started Guide! `Join our forum <https://discuss.daml.com>`_ and share a screenshot of your accomplishment to `get your second of 3 badges <https://discuss.daml.com/badges/126/hey-look-what-i-can-do>`_! Get the third badge by `deploying to project:DABL <https://daml.com/learn/getting-started/deploy-to-dabl/>`_
+
```

Comment:
Third badge is missing but I'm pretty sure you're working on it 😄

Reply:
You can only get the third badge with daml.com/learn so I linked to the task where you can get it.

Reply:
Ah, I see. Makes sense.